.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,39 @@
+name: Scorecard security
+on:
+  branch_protection_rule:
+  schedule:
+    - cron: "43 5 * * 0"
```

Comment:
Not really. This scorecard is more of a repository related security evaluation then code evaluation. So it only make sense to run on `main`. And this `on:` field works as `or`. So this will be run on every push to `main` and on every change to branch protection settings and additionally every Sunday.

Reply:
If I make a change that lowers our score shouldn't I be able to know that during pull request review? Why postpone the check after the code is already on main?

Reply:
I think it would be cumbersome to manually review results of this tool for every PR. It contain some checks that are really repository dependent, e.g. it checks history of the branch to calculate the score, so you not always have direct influence on the score. So because of this I don't think it makes sense to run it no branches. Also this seems to be recommended way to run it